modules/serverless-logic-workflow-changing-base-builder-image.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-logic/serverless-logic-creating-managing-workflows.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-workflow-changing-base-builder-image"]
+= Changing the base builder image
+
+You can modify the default builder image used by the {ServerlessLogicOperatorName} by editing the `sonataflow-operator-builder-config` ConfigMap.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* Install the OpenShift CLI `(oc)`.
+
+.Procedure
+
+. Edit the `sonataflow-operator-builder-config` ConfigMap.
+[source,terminal]
+----
+$ oc edit cm/sonataflow-operator-builder-config -n sonataflow-operator-system
+----
+
+. Modify the dockerfile entry.
++
+In your editor, locate the dockerfile entry and change the first line to the desired image.
++
+.Example
+[source,yaml]
+----
+data:
+  Dockerfile: |
+    FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.32.0
```

Comment:
```suggestion
    FROM registry.redhat.io/openshift-serverless-1-tech-preview/logic-data-index-ephemeral-rhel8:1.32.0
```

Can you replace `1.32.0` with a variable to reflect the current version?

Reply:
This is not 100% fine, use mine suggestion below please. We must use the builder related image, instead of the data-index related.

Reply:
Okay, noted!